frontend/src/screens/alby/SupportAlby.tsx

```diff
@@ -155,208 +156,187 @@ function SupportAlby() {
-                            <div className="grid grid-cols-3 gap-1 mt-1">
-                              <Button
-                                type="button"
-                                variant="outline"
-                                onClick={() => setAmount("3000")}
-                              >
-                                🙏 3000
-                              </Button>
-                              <Button
-                                type="button"
-                                variant="outline"
-                                onClick={() => setAmount("6000")}
-                              >
-                                💪 6000
-                              </Button>
-                              <Button
-                                type="button"
-                                variant="outline"
-                                onClick={() => setAmount("10000")}
-                              >
-                                ✨ 10000
-                              </Button>
-                            </div>
-                          </div>
-                        </div>
-                        <div className="grid grid-cols-4 items-center gap-4">
-                          <Label htmlFor="comment" className="text-right">
-                            Name{" "}
-                            <span className="font-normal text-muted-foreground">
-                              (optional)
-                            </span>
-                          </Label>
-                          <Input
-                            id="sender-name"
-                            value={senderName}
-                            onChange={(e) => setSenderName(e.target.value)}
-                            placeholder={`Nickname, npub, @twitter, etc.`}
-                            className="col-span-3"
-                          />
+      <AppHeader
+        title="Support Alby Hub"
+        description="We are committed to elevating the Bitcoin ecosystem by offering reliable, efficient, and user-friendly software solutions for seamless transactions. With your help, we can keep pushing boundaries and evolving Alby Hub into something extraordinary."
+      />
+      <h2 className="text-2xl font-semibold">Become a Supporter</h2>
+      <div className="grid grid-cols-1 sm:grid-cols-2 gap-4">
+        <Card className="flex flex-col">
+          <CardHeader className="grow">
+            <CardTitle>Upgrade to Pro</CardTitle>
+            <CardDescription>
+              Upgrade your Alby Account to Pro for a small fee and enjoy
+              additional perks that come with it!
+            </CardDescription>
+          </CardHeader>
+          <CardFooter className="flex justify-end">
+            <UpgradeDialog>
+              <Button>
+                <Sparkles className="w-4 h-4 mr-2" />
+                Upgrade to Pro
+              </Button>
+            </UpgradeDialog>
+          </CardFooter>
+        </Card>
```

Comment:
We show this even when the user is subscribed to Pro, I check if we don't redirect but even in the sidebar there's no check to see if they are already subscribed

Reply:
In the sidebar there is a check: 

https://github.com/getAlby/hub//blob/d4bf1cddf53229ca144162dde7a28ae667d8d9de/frontend/src/components/SidebarHint.tsx#L88-L91

Reply:
Ah sorry missed that :+1: